index.html

```diff
@@ -7,5 +7,6 @@
 <title>Welcome to Potionfy</title>
 </head>
 <body>
+  <h1>Welcome to los Angels ! Enjoy your stay</h1>                                                                 
```

Comment:
Could you remove 'Enjoy your stay' string and make the 'Welcome to Los Angels' string red?